src/libs/ReportActionsUtils.ts

```diff
@@ -1,8 +1,8 @@
+import fastMerge from 'expensify-common/lib/fastMerge';
```

Comment:
I'm getting a type error from this line:

<img width="1197" alt="image" src="https://github.com/Expensify/App/assets/87341702/f0b99f05-3c68-4416-bc83-8b033ceca009">

<img width="788" alt="image" src="https://github.com/Expensify/App/assets/87341702/a5d6d2aa-1e45-448b-a639-d85806cf798c">

Does anyone know how to fix it?

Reply:
Oh, it got fixed after deleting `node_modules` and running `npm install` again